src/argilla/training/base.py

```diff
@@ -261,6 +292,8 @@ def save(self, output_dir: str):
 class ArgillaTrainerSkeleton(ABC):
     def __init__(
         self,
+        name: str,
+        workspace: str,
```

Comment:
I guess we can make this optional and default to `rg.get_workspace()`?

```suggestion
        workspace: Optional[str] = rg.get_workspace(),
```

Or the following instead:

```suggestion
        workspace: Optional[str] = None,
```

And then

```python
if workspace is None:
    workspace = rg.get_workspace()
...
```

WDYT? @davidberenstein1957